Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -157,14 +182,46 @@ public static string GetCacheFileName(Uri uri)
                 }
                 else
                 {
-                    busy = GetFromCacheOrDownloadAsync(uri);
+                    busy = new ConcurrentTask()
```

Comment:
It was called busy before I started working on this class. Renamed it in the latest commit.